os/os.h

```diff
@@ -23,7 +23,7 @@ enum {
 	os_windows,
 	os_android,
 	os_dragonfly,
-
```

Comment:
The space was most likely thought to separate the os_nr from the rest. So maybe put it below the new os_rtems.